### ISSUE TITLE
[launchpad] if loading the first page in Firefox fails, the server throws

### ISSUE THREAD

**Issue by julienw** [devtools-html/devtools-core#222]
STR:
1. Disconnect the network
2. click "launch firefox"

=> Because we have no network, the intended page doesn't load and the server is crashing.

I expect the server to not crash so that I can (for example) debug a local page.

It's very common that I work in a no-network condition so I really want to see this fixed. I tried quickly adding a try/catch in bin/firefox-driver but either I couldn't find the proper location or it doesn't work like that.

**Comment by irfanhudda** [comment]
I usually run `debugger-examples` locally by editing [https://github.com/devtools-html/debugger.html/blob/master/package.json#L29](https://github.com/devtools-html/debugger.html/blob/master/package.json#L29) and replacing `https://devtools-html.github.io/debugger-examples/` with `http://localhost:5678`.

**Comment by julienw** [comment]
yeah, I used `about:home` too but it shuoldn't stop the server :)
